bigchaindb/tendermint/lib.py

```diff
@@ -243,5 +243,18 @@ def validate_transaction(self, tx, current_transactions=[]):
     def fastquery(self):
         return fastquery.FastQuery(self.connection, self.me)
 
+    def get_validators(self):
+        try:
+            resp = requests.get(f'{ENDPOINT}validators')
+            validators = resp.json()['result']['validators']
+            for v in validators:
+                v.pop('accum')
```

Comment:
Just a question: Why are you removing these keys? What is there significance?

Reply:
Since we query Tendermint for the validator set, we just clean up the response a little and serve just the necessary data. The usual response looks something as below,

```json
{
  "jsonrpc": "2.0",
  "id": "",
  "result": {
    "block_height": 1,
    "validators": [
      {
        "address": "6FEE524CE1E29425EF369CA63FFB1A9204EB0F12",
        "pub_key": {
          "type": "ed25519",
          "data": "B0E42D2589A455EAD339A035D6CE1C8C3E25863F268120AA0162AD7D003A4014"
        },
        "voting_power": 10,
        "accum": -10
      },
      {
        "address": "F5426F0980E36E03044F74DD414248D29ABCBDB2",
        "pub_key": {
          "type": "ed25519",
          "data": "4E2685D9016126864733225BE00F005515200727FBAB1312FC78C8B76831255A"
        },
        "voting_power": 10,
        "accum": 10
      }
    ]
  }
}
```